pages/common/sed.md

```diff
@@ -1,36 +1,33 @@
 # sed
 
 > Edit text in a scriptable manner.
-> More information: <https://man.archlinux.org/man/sed.1>.
+> See also: `awk`, `ed`.
+> More information: <https://manned.org/sed.1>.
 
-- Replace the first occurrence of a regular expression in each line of a file, and print the result:
+- Execute commands:
 
-`sed 's/{{regular_expression}}/{{replace}}/' {{filename}}`
+`sed '{{s/apple/mango/g}}'`
 
-- Replace all occurrences of an extended regular expression in a file, and print the result:
+- Execute a script:
 
-`sed -r 's/{{regular_expression}}/{{replace}}/g' {{filename}}`
+`sed --file={{path/to/script.sed}}`
 
-- Replace all occurrences of a string in a file, overwriting the file (i.e. in-place):
+- Execute commands with enabled extended regular expressions:
 
-`sed -i 's/{{find}}/{{replace}}/g' {{filename}}`
+`sed --regexp-extended '{{s/(apple)/\U\1/g}}'`
 
-- Replace only on lines matching the line pattern:
+- Execute commands without automatic buffer printing:
 
-`sed '/{{line_pattern}}/s/{{find}}/{{replace}}/' {{filename}}`
+`sed --quiet '{{1p}}'`
 
-- Delete lines matching the line pattern:
+- Execute commands and replace file with it's output:
```

Comment:
Perhaps this will read better? Also, `expression` is more technically correct than `command`.
```suggestion
- Execute an expression and replace a file with it's output:
```